HW5/task1.py

```diff
@@ -6,18 +6,23 @@ def add_contact():
     global name_lastname
     global pnumber
     while True:
-        name_lastname = input("Name: ")
-        pnumber = input("Phone number: +380")
-        if name_lastname in contacts:
-            print("You already have this contact saved")
-        elif len(pnumber) == 9:
-            print(f"{name_lastname} contact is saved")
-        elif len(pnumber) < 9 > len(pnumber):
-            print("Your number should be 9 digits long")
-            pnumber = input("Phone number: +380")
-            print(f"{name_lastname} contact is saved")
-        contacts[name_lastname] = pnumber
-        menu()
+        try:
+            name_lastname = input("Name: ")
+            pnumber = int(input("Phone number: +380"))
+        except ValueError:
+            print("Please use only digits")
+            # how to make the program go back to the beginning of the while loop?
+            if name_lastname in contacts:
```

Comment:
I think this block should be in `else` block or out of try...except constraction